[2021-06-27 18:01:35,360] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.export_postgresql_customer 2021-06-27T18:01:27.549223+00:00 [queued]>
[2021-06-27 18:01:35,401] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.export_postgresql_customer 2021-06-27T18:01:27.549223+00:00 [queued]>
[2021-06-27 18:01:35,402] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 18:01:35,402] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-06-27 18:01:35,406] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 18:01:35,433] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): export_postgresql_customer> on 2021-06-27T18:01:27.549223+00:00
[2021-06-27 18:01:35,441] {standard_task_runner.py:52} INFO - Started process 7054 to run task
[2021-06-27 18:01:35,454] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgretomysql', 'export_postgresql_customer', '2021-06-27T18:01:27.549223+00:00', '--job-id', '83', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/postgresql-collect.py', '--cfg-path', '/tmp/tmpjibo2lat', '--error-file', '/tmp/tmpo4labnaq']
[2021-06-27 18:01:35,461] {standard_task_runner.py:77} INFO - Job 83: Subtask export_postgresql_customer
[2021-06-27 18:01:35,580] {logging_mixin.py:104} INFO - Running <TaskInstance: postgretomysql.export_postgresql_customer 2021-06-27T18:01:27.549223+00:00 [running]> on host b69deda3a989
[2021-06-27 18:01:35,660] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=tech@innospark.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgretomysql
AIRFLOW_CTX_TASK_ID=export_postgresql_customer
AIRFLOW_CTX_EXECUTION_DATE=2021-06-27T18:01:27.549223+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-27T18:01:27.549223+00:00
[2021-06-27 18:01:35,662] {logging_mixin.py:104} INFO - Entering export_postgresql_to_csv SELECT * FROM test.customer
[2021-06-27 18:01:35,679] {base.py:78} INFO - Using connection to: id: docker-postgres. Host: 172.27.240.1, Port: 5432, Schema: , Login: postgres, Password: ***, extra: {}
[2021-06-27 18:01:40,497] {base.py:78} INFO - Using connection to: id: docker-postgres. Host: 172.27.240.1, Port: 5432, Schema: , Login: postgres, Password: ***, extra: {}
[2021-06-27 18:01:40,527] {python.py:151} INFO - Done. Returned value was: None
[2021-06-27 18:01:40,567] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=postgretomysql, task_id=export_postgresql_customer, execution_date=20210627T180127, start_date=20210627T180135, end_date=20210627T180140
[2021-06-27 18:01:40,649] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-27 18:01:40,744] {local_task_job.py:151} INFO - Task exited with return code 0
